customize.dist/messages.js

```diff
@@ -118,6 +118,10 @@ define(req, function(AppConfig, Default, Language) {
 
     Messages._languages = map;
     Messages._languageUsed = language;
+    Messages.admin_logoSize_error = "The logo size must be smaller than 200KB"; // XXX
```

Comment:
The value "200KB" should be passed from the code as a value, with the placeholder "{0}" in the translated string